ci/build_common.sh

```diff
@@ -89,6 +89,8 @@ echo "CXX_STANDARD=$CXX_STANDARD"
 echo "GPU_ARCHS=$GPU_ARCHS"
 echo "PARALLEL_LEVEL=$PARALLEL_LEVEL"
 echo "BUILD_DIR=$BUILD_DIR"
+echo "Current commit is:"
+git show --pretty=oneline --abbrev-commit
```

Comment:
This will currently show the full diff of the most recent commit, which can be extremely verbose. How about we just show the commit hash, date, and commit message? 

```suggestion
git log -1
```

Reply:
I thought that it did that but log -1 is definitely clearer